src/lexedata/edit/add_concepticon.py

```diff
@@ -126,10 +126,37 @@ def add_concepticon_references(
     dataset.write(ParameterTable=write_back)
 
 
+def add_concepticon_definitions(
+    dataset: pycldf.Dataset,
+    column_name: str = "Concepticon_Definition",
+):
+    # Create a concepticon_definition column
+    try:
+        dataset.add_columns("ParameterTable", column_name)
+        dataset.write_metadata()
+    except ValueError:
+        raise ValueError(
+            f"{column_name} could not be added to ParameterTable of {dataset}."
+        )
+    # write concepticon definitions
+    write_back = []
+    for row in dataset["ParameterTable"]:
+        try:
+            row[column_name] = concepticon.api.conceptsets[
+                row[dataset.column_names.parameters.concepticonReference]
```

Comment:
The “KeyError: `row` has no key None” thrown in this case if there is no concepticonReference will look very opaque.
Let's check for its existence as the first thing in the function.